create_jenkins_job.py

```diff
@@ -172,7 +172,7 @@ def main(argv=None):
             job_name = job_name[0:-5]
         job_data['time_trigger_spec'] = '0 12 * * *'
         job_data['cmake_build_type'] = 'None'
-        job_data['ament_test_args_default'] = '--retest-until-fail 20 -LE linter'
+        job_data['ament_test_args_default'] = '--retest-until-fail 20 --ctest-args -LE linter --'
```

Comment:
I think what's happening here is that the `--ctest-args` (greedy) are being nested within another greedy argument `--ament-test-args` that gets the delimeter appended [here](https://github.com/ros2/ci/blob/9923cb7ae768dcb56007f5e45cd0c1ec69819795/job_templates/ci_job.xml.template#L148). Really these two lists should have different variables.

As you mentioned offline, this will work as-is for now, so we can apply it to the farm, but let's not merge it until it's implemented in a way that's more future-proof